scripts/install-reqs.sh

```diff
@@ -39,5 +39,27 @@ install_gometalinter() {
   rm -f "$GOBINDIR/$LINTER_TARBALL"
 }
 
+install_etcd() {
```

Comment:
We'd removed this! When?

Also, @sseshasa will be bumping up etcd version in Gopkg.toml in #1068.

Reply:
I"m not sure if it was present as elastic etcd was the default.